airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py

```diff
@@ -177,6 +196,19 @@ def test_additional_properties_is_true(self, actual_connector_spec):
                 [additional_properties_value is True for additional_properties_value in additional_properties_values]
             ), "When set, additionalProperties field value must be true for backward compatibility."
 
+    @pytest.mark.default_timeout(60)  # Pulling the previous connector image can take more than 10 sec.
+    @pytest.mark.spec_backward_compatibility
```

Comment:
Was this `pytest.mark.spec_backward_compatibility` left over from the previous skipping logic or do we still want this here?

Reply:
I was thinking it could be useful to keep the marker to keep an alternate way of bypassing this test suite.